dist/tools/has_higher_version/has_higher_version.sh

```diff
@@ -0,0 +1,33 @@
+#! /bin/bash
+#
+# usage: has_higher_version.sh <version> <minimal_version> [toolname]
+#   Checks that version >= minimal_version
+#   Version format MAJOR.MINOR.PATCH ex 3.1.4
+
+# Adapted from https://stackoverflow.com/a/24067243 to have >=
+# Checking versions with '-rcX' does not work properly due to the usage of
+# `sort -V`.
+
+if [ $# -lt 2 ]; then
+    echo "usage: ${0} <version> <minimal_version> [toolname]" >&2
+    echo "  Checks that version >= minimal_version" >&2
+    echo "  Version format MAJOR.MINOR.PATCH ex 3.1.4" >&2
+    exit 1
+fi
+
+readonly TOOLVERSION="${1}"
+readonly MINVERSION="${2}"
+readonly TOOLNAME="${3}"
+
+if [ "${TOOLNAME}" != "" ]; then
+  readonly TOOLSTR="${TOOLNAME} "
+else
+  readonly TOOLSTR=""
+fi
+
+
+HIGHEST=$(printf "%s\n%s\n" "${TOOLVERSION}" "${MINVERSION}" | sort -rV | head -n 1)
+test "${HIGHEST}" = "${TOOLVERSION}" && exit 0
+
+printf "\n%sversion %s is not >= to minimal required %s\n\n" "${TOOLSTR}" "${TOOLVERSION}" "${MINVERSION}"
```

Comment:
maybe rephrase: `printf "\nInvalid version, found %s%s, minimal required is %s\n\n" "${TOOLSTR}" "${TOOLVERSION}" "${MINVERSION}"`

The `>=` should be implied by *minimal required*.

Reply:
Will do, I should also print to stderr.